guide/daily-spending-wallet/landing-page.md

```diff
@@ -30,71 +30,189 @@ Illustration source:
 
 # Daily spending wallet
 
-This section is an in-depth design exploration of a bitcoin wallet for on-the-go payments. It focuses are on:
+The daily spending wallet is an app designed to quickly and easily send small amounts of bitcoin. It focuses on the following:
+
+<div class="center product-feature" markdown="1">
+   {% include picture.html
+      image = "/assets/images/guide/daily-spending-wallet/bitcoin.png"
+      retina = "/assets/images/guide/daily-spending-wallet/bitcoin@2x.png"
+      mobile = "/assets/images/guide/daily-spending-wallet/bitcoin-mobile.png"
+      mobileRetina = "/assets/images/guide/daily-spending-wallet/bitcoin-mobile@2x.png"
+      alt-text = "Bitcoin logo"
+      width = 100
+      height = 100
+   %}
+
+   <div>
+      <h2 class="h4">Bitcoin only</h2>
+      <p>This wallet aims to provide the best experience possible for using a single currency: bitcoin.</p>
+   </div>
+</div>
+
+<div class="center product-feature" markdown="1">
+   {% include picture.html
+      image = "/assets/images/guide/daily-spending-wallet/private-key.png"
+      retina = "/assets/images/guide/daily-spending-wallet/private-key@2x.png"
+      mobile = "/assets/images/guide/daily-spending-wallet/private-key-mobile.png"
+      mobileRetina = "/assets/images/guide/daily-spending-wallet/private-key-mobile@2x.png"
+      alt-text = "Graphic of a key"
```

Comment:
No need to include the word "Graphic" in alt-text. Overall, you may not even need alt-text for these 4 graphics, they don't really add much information that is not already captured in the copy right next to them.

Reply:
Done